core/deployment/src/main/java/io/quarkus/deployment/images/ContainerImages.java

```diff
@@ -57,6 +57,11 @@ public class ContainerImages {
     public static final String QUARKUS_MICRO_VERSION = "2.0";
     public static final String QUARKUS_MICRO_IMAGE = QUARKUS_MICRO_IMAGE_NAME + ":" + QUARKUS_MICRO_VERSION;
```

Comment:
I think they should be adjusted with the `UBI8_` prefix. We should also make sure they are not used in the codebase.

Reply:
Done. I have checked for all occurrences of `quarkus-micro-image` and I found one in docs that I think is incorrect, though it may be intentional. Will let you decide :shrug:

Reply:
Where did you find it? I had a look and I only find references to `ubi9-quarkus-micro-image`.

Reply:
Sorry for the confusion, I was referring to adding `9` in docs/src/main/asciidoc/quarkus-runtime-base-image.adoc. I kinda thought maybe Clement wanted it to be not-specific to some UBI version, but IMHO it is confusing.